packages/node/src/client.ts

```diff
@@ -31,6 +31,7 @@ export default class LogtoNodeBaseClient extends BaseClient {
     resource,
     fetchUserInfo,
     getOrganizationToken,
+    accessTokenOrganizationId,
```

Comment:
Would it be OK to just use `organizationId`?

Reply:
To avoid confusing with "organization token".

Reply:
Per discussion, changed to `organizationId`, we will deprecate `getOrganizationToken` later.